test/math/FloatPoint2DTest.ooc

```diff
@@ -49,8 +50,8 @@ FloatPoint2DTest: class extends Fixture {
 		this add("casting", func {
 			value := "10.00, 20.00"
 			expect(this point3 toString(), is equal to(value))
-			expect((FloatPoint2D parse(value)) x, is equal to((this point3) x))
-			expect((FloatPoint2D parse(value)) y, is equal to((this point3) y))
+			expect((FloatPoint2D parse(Text new(value))) x, is equal to((this point3) x))
+			expect((FloatPoint2D parse(Text new(value))) y, is equal to((this point3) y))
```

Comment:
Sure, that works too, but we'll be revisiting this soon enough when `toText()` is introduced and `toString()` finally deprecated and removed. So, do I need to fix it now?